### PR TITLE
Store the projects and jobs to datastore when pressing 'Fetch' button

### PR DESCRIPTION
**Pull request by andreeanica16**
1. Add button on the html that triggers fetching the data and storing it on Datastore
2. Add class JobStoreCenter that deals with all the interactions with Datastore. It has the functionality to add a new Poject with all its jobs.
3. Add JobServlet that receives the command from the user to start the fetching and storing process.

### CODE REVIEW

**Comment by andreeanica16** [comment]
@iht @juliocc @tblanshard

**Review by juliocc — APPROVED** [review]
LGTM

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -6,7 +6,8 @@
  */
 
 function initBody() {
-  checkPermissions();
+    fetch('/data').then(console.log('Works'));
```
Comment (tblanshard) [inline src/main/webapp/script.js:9]: I think this isn't needed anymore as isn't it calling the test servlet?
```suggestion
```

**Review by tblanshard — DISMISSED** [review]
LGTM

File: src/main/webapp/script.js (hunk 1 above)
Comment (andreeanica16) [reply src/main/webapp/script.js:9]: You are right! :D

**Review by tblanshard — APPROVED** [review]
LGTM :)

**Pull request merged**